src/test/java/tdl/datapoint/infra_events/AWSEventsDatapointAcceptanceTest.java

```diff
@@ -103,14 +104,35 @@ public void start_recorder() throws Exception {
                 convertToMap(wrapAsSNSEvent(s3PutEvent)),
                 NO_CONTEXT);
 
-        // Then - Repo is created with the contents of the Video or SRCS file
+        // Then - Object hits the S3 bucket triggering the S3 put event
         waitForQueueToReceiveEvents();
         RecorderStartedEvent queueEvent = recorderStartedEvents.pop();
         String eventString = queueEvent.toString();  // eventually might be a idea to verify the event sent getEventAsJsonString();
         assertThat(eventString, allOf(containsString(participantId),
                 containsString(challengeId)));
     }
 
+    @Test
+    public void start_something_unsupported_no_event_should_flow_through_the_sqs_server() throws Exception {
```

Comment:
I think there are two behaviours here:
- unrecognised events should not flow to SQS
- unrecognised events should raise exception